src/style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300px;
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
```

Comment:
there is no such font-family, you've described only Avenir + its two font-weights, so where Avenir-Book is supposed to come from? right – it won't happen, so that you have to use Avenir with normal font-weight